sdk/tests/client/input_selection/foundry_outputs.rs

```diff
@@ -107,7 +129,7 @@ fn minted_native_tokens_in_new_remainder() {
     let alias_id_2 = AliasId::from_str(ALIAS_ID_2).unwrap();
 
     let inputs = build_inputs(vec![
-        Basic(1_000_000, BECH32_ADDRESS_ED25519_0, None, None, None, None, None, None),
+        Basic(2_000_000, BECH32_ADDRESS_ED25519_0, None, None, None, None, None, None),
```

Comment:
Why?

Reply:
Probably updated because it was necessary somewhere else, changed back